app/helpers/actionTicketManager/index.tsx

```diff
@@ -0,0 +1,182 @@
+import axios from "axios";
+import * as uuid from "uuid/v4";
+import * as store from "store";
+import * as expirePlugin from "store/plugins/expire";
+import EnvChecker from "../envChecker";
+import ActionTicket from "./actionTicket";
+
+export const MAXIMUM_TICKET_COUNT_IN_QUEUE = 5;
+const TIME_INTERVAL_TO_SEND_TICKETS = 1000 * 5;
+export const DEVICE_ID_KEY = "d_id";
+export const SESSION_ID_KEY = "s_id";
+export const TICKET_QUEUE_KEY = "a_q";
+export const DEAD_LETTER_QUEUE_KEY = "d_a_q";
+const LIVE_SESSION_LENGTH = 1000 * 60 * 30;
+const MAXIMUM_RETRY_COUNT = 3;
+const DESTINATION_URL = "https://gxazpbvvy7.execute-api.us-east-1.amazonaws.com/dev/actionticket";
+// const DESTINATION_URL = "http://localhost:3000";
+
+export type Ticket = FinalActionTicket & ActionTicketMeta;
+export interface FinalActionTicket extends ActionTicketParams {
+  deviceId: string;
+  sessionId: string;
+  createdAt: string;
+  userId: string | null;
+}
+
+interface ActionTicketMeta {
+  errorCount?: number;
+}
+
+export interface ActionTicketParams {
```

Comment:
In order to use ActionTicket as a class, I think interface naming convention should be unified in the purpose of compromising.
For example, ActionTicketParams -> IBaseActionTicket, FinalActionTicket -> IActionTicket
(like "**_Java_**" way)